causalpy/pymc_models.py

```diff
@@ -11,6 +25,13 @@
 class ModelBuilder(pm.Model):
     """
     This is a wrapper around pm.Model to give scikit-learn like API.
+
+    Public Methods
+    ---------------
+    - build_model: must be implemented by subclasses
+    - fit: populates idata attribute
+    - predict: returns predictions on new data
+    - score: returns Bayesian R^2
```

Comment:
We should be able get the R^2 rendered as math, https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-math